Client/Frontend/Theme/Themable.swift

```diff
@@ -4,21 +4,7 @@
 
 import Foundation
 
-protocol Themeable: Notifiable {
+protocol Themeable: UIViewController {
```

Comment:
This can't conform to Notifiable because it references structs and Notifiable needs to be readable by objective-c 😞